frontend/src/metabase/modes/components/drill/ObjectDetailDrill.jsx

```diff
@@ -96,5 +83,8 @@ export default ({ question, clicked }) => {
   const { column, value: objectId, extraData } = clicked;
   const params = { question, column, objectId, extraData };
   const actionObject = isPK(column) ? getPKAction(params) : getFKAction(params);
+  if (!hasManyPKColumns(question)) {
+    actionObject.zoomInRow = objectId;
+  }
```

Comment:
1) I don't think so, since currently we're always fitlering to a single row for FK drills
2) I think composite key edge cases are better handled in this PR: https://github.com/metabase/metabase/pull/22398, which greatly simplifies viewing them

Reply:
Should we use the `getExtraData` approach then as it used to be until the next PR is merged?